docs/configuration_file.rst

```diff
@@ -1,22 +1,1167 @@
-==========================
-Configuration file example
-==========================
+  
+==================
+Configuration file
+==================
 
-The configuration files follows the `YAML standard <https://yaml.org/>`_.
+The configuration file contains all information needed to build, run and analyse the results from the FlowNet network model.
+
+name
+====
+
+
+
+
+flownet
+=======
+
+A description of the usage of the various keywords available in relation to the construction of the FlowNet network model. 
+These keywords are mainly related to the geometry of the model, populating with model parameters will be described under 
+`model_parameters`_. For simplicity the keywords are listed alphabetically (by indentation), they can appear in any order 
+in the actual configuration file.
+
```

Comment:
Should it be mentioned which unit convention is used? For example, there are pressure inputs for a few options, as well as the `min_permeability` option (although here it is explicitly said that it should be in mD).

Reply:
I added information on how the unit conversion can potentially be changed. Don't think it should be necessary with a full list of units.